### PR TITLE
^adapting for new structure, adding good shit to array

### PR DESCRIPTION
**Pull request by thekanbo**

### CODE REVIEW

File: handlers/goodshit.js [hunk 1]
```diff
@@ -18,23 +18,38 @@ var GOOD_SHIT_ARRAY = [
...
+  'oh👌fuck👌',
+  'oh👌shit👌',
+  'fuck👌yes👌',
+  '(socrates died for real)'
+  ];
 
-function goodshit(client, nick, to, text, message, params, buffer) {
-  var msg = _.sample(buffer);
+function goodshit(channel, message, params) {
+  LOG.info(params);
```
Comment (skeltonath) [inline handlers/goodshit.js]: probably don't need to log all the params

File: handlers/goodshit.js [hunk 2]
```diff
@@ -18,23 +18,38 @@ var GOOD_SHIT_ARRAY = [
...
+  'right✔there',
+  'right👌there',
+  'mMMMMᎷМ',
+  '🍕🍅🍕🍅 ',
+  'sign me the fuck up',
+  'oh👌fuck👌',
+  'oh👌shit👌',
+  'fuck👌yes👌',
+  '(socrates died for real)'
+  ];
```
Comment (skeltonath) [inline handlers/goodshit.js]: indentation is off

File: handlers/goodshit.js [hunk 3]
```diff
@@ -18,23 +18,38 @@ var GOOD_SHIT_ARRAY = [
...
 
-function goodshit(client, nick, to, text, message, params, buffer) {
-  var msg = _.sample(buffer);
+function goodshit(channel, message, params) {
+  LOG.info(params);
+  var msg = params;
```
Comment (skeltonath) [inline handlers/goodshit.js]: use `let` instead of `var`

File: handlers/goodshit.js [hunk 4]
```diff
@@ -1,16 +1,16 @@
-var _      = require('lodash');
-var log4js = require('log4js');
-var format = require('format');
+let _      = require('lodash');
+let log4js = require('log4js');
+let format = require('format');
 
-var LOG = log4js.getLogger('goodshit');
+let LOG = log4js.getLogger('goodshit');
```
Comment (skeltonath) [inline handlers/goodshit.js]: oh, missed this earlier

for imports and constants use `const` instead of `let`

File: handlers/goodshit.js [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
 
 module.exports = {
   name: 'goodshit',
   command: 'gs',
   handler: goodshit
 };
 
-var GOOD_SHIT_ARRAY = [
+let GOOD_SHIT_ARRAY = [
```
Comment (skeltonath) [inline handlers/goodshit.js]: `const`

File: handlers/goodshit.js [hunk 6]
```diff
@@ -36,35 +36,34 @@ var GOOD_SHIT_ARRAY = [
   ];
```
Comment (skeltonath) [inline handlers/goodshit.js]: indentation still off right here

bring that `];` back haha

**Comment by skeltonath** [comment]
Testing webhooook

**Comment by thekanbo** [comment]
Merge my changes you cowards

File: handlers/goodshit.js [hunk 7]
```diff
@@ -1,55 +1,106 @@
-var _      = require('lodash');
-var log4js = require('log4js');
-var format = require('format');
+const _			= require('lodash');
```
Comment (skeltonath) [inline handlers/goodshit.js]: holy shit lol

File: handlers/goodshit.js [hunk 8]
```diff
@@ -1,55 +1,106 @@
...
+const format = require('format');
 
-var LOG = log4js.getLogger('goodshit');
+const LOG = log4js.getLogger('goodshit');
 
 module.exports = {
-  name: 'goodshit',
-  command: ':gs',
-  handler: goodshit
+	name: 'goodshit',
```
Comment (skeltonath) [inline handlers/goodshit.js]: what kind of goddamn circus you running with TABS

we are a spaces only kind of repo

**Comment by thekanbo** [comment]
release me

**Pull request merged**